src/harness/reference_models/iap/iap.py

```diff
@@ -134,12 +128,8 @@ def iapPointConstraint(protection_point, channels, low_freq, high_freq,
       protection_constraint)
 
   if not neighbor_grants:
-    for j in xrange(len(channels)):
-      asas_interference.UpdateAggregateInterferenceInfo(
-          protection_point[1], protection_point[0], 0)
-      aggr_interference.UpdateAggregateInterferenceInfo(
-          protection_point[1], protection_point[0], 0)
-    return
+    return (protection_point[1], protection_point[0],
```

Comment:
I understand that you are trying to return the original asas_interference and aggr_interference here and all other places in stead of updating them. I will have to true you that they are logically correct and not affect the calculation results. If you test results are correct, I have no objections, but do please make sure tested and verified.

Reply:
Yes I checked carefully and compared the examples before and after. They give exactly the same results on all 15 decimals printed